src/components/productdisplayitem.main.jsx

```diff
@@ -418,7 +419,11 @@ class ProductDisplayItemMain extends React.Component {
       }
       let availability = (productData._addtocartform[0].links.length > 0);
       let availabilityString = '';
+      let productLink = '';
       if (productData._availability.length >= 0) {
+        if (productData._code) {
+          productLink = `https://reference.elasticpath.com/itemdetail/${productData._code[0].code}`;
```

Comment:
We can't hardcode the domain for this link. Need to use window.location or hostname to grab it.